common/src/health.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Health
+ *
+ * Create an endpoint on /health to check all dependent
+ * resources
+ */
+
+const { createTerminus, HealthCheckError } = require('@godaddy/terminus');
+
+const { promiseTimeout } = require('./utils');
+
+module.exports = ({
+  server,
+  tasks,
+  logger,
+  timeout = 1000,
+  onTerminate = null,
+  terminationGrace = 0,
+}) =>
+  createTerminus(server, {
+    signal: 'SIGINT',
+    async beforeShutdown() {
+      if (terminationGrace > 0) {
+        logger.info({ terminationGrace }, 'Pausing before shutdown');
+
+        await new Promise((resolve) => setTimeout(resolve, terminationGrace));
+      }
+    },
+    healthChecks: {
+      '/health': async () => {
+        const startTime = Date.now();
+
+        const results = (
+          await Promise.allSettled(
```

Comment:
til: the result of `.allSettled` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled

could not work out where `value` was coming from on line 51